src/hotspot/share/runtime/continuationJavaClasses.hpp

```diff
@@ -107,13 +107,15 @@ class jdk_internal_vm_StackChunk: AllStatic {
   static inline int size(oop chunk);
   static inline void set_size(HeapWord* chunk, int value);
 
+  static inline void set_bottom(HeapWord* chunk, int value);
```

Comment:
Shouldn't this be moved down to the other set_bottom? The two set_sp functions are held together.